mumlib/src/command.rs

```diff
@@ -1,6 +1,59 @@
 use crate::state::{Channel, Server};
 
+use chrono::NaiveDateTime;
 use serde::{Deserialize, Serialize};
+use std::fmt;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct MumbleEvent {
```

Comment:
As above, I think that a policy that new structs also should have doc comments to work on documenting the codebase.